Cargo.toml

```diff
@@ -36,17 +36,16 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw0 = {  version = "0.10.0" }
-cw2 = { version = "0.11" }
-cw20 = { version = "0.10.0" }
-cw20-base = { version = "0.10.0", features = ["library"] }
-cosmwasm-std = { version = "1.0.0-beta" }
-cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.10"
-schemars = "0.8.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.26" }
+cw0 = "0.10.3"
```

Comment:
how would you feel about removing this dependency, adding one for `cw_utils = "0.15.0"` and using `cw_utls::parse_reply_instantiate_data` in the reply handler? i think this would fit the CosmWasm 1.1 way best. :)

Reply:
sure :) 2d301dff3d9cd62b637bea2a13ac5b599664c7bb